scss/style.scss

```
@@ -0,0 +1,11 @@
+@import "_header.scss";
```

Такі імпорти можна писати просто
```suggestion
@import "header";
```